src/applications/edu-benefits/0994/migrations/version2.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+
+export function version2() {
+  return savedData => {
+    const trainingProgramsChoice = _.get(
+      savedData.metadata,
+      'view:trainingProgramsChoice',
+    );
+
+    if (trainingProgramsChoice) {
+      const newData = _.set(
+        savedData.metadata,
+        'hasSelectedPrograms',
+        trainingProgramsChoice,
+      );
```

Comment:
Just to be safe, you should also delete the old field after:

`delete newData['view:trainingProgramsChoice']`.